src/vivarium_conic_lsff/components/fortification/folic_acid.py

```diff
@@ -192,11 +192,57 @@ def load_population_attributable_fraction_data(builder: 'Builder'):
         rr = params.sample_folic_acid_relative_risk(location, draw)
         coverage = params.sample_folic_acid_coverage(location, draw, 'baseline')
         exposure = 1 - coverage
-        mean_rr = rr*exposure + 1*(1-exposure)
-        paf = (mean_rr - 1)/mean_rr
+        mean_rr = rr * exposure + 1 * (1 - exposure)
+        paf = (mean_rr - 1) / mean_rr
         return paf
 
 
+class MaternalIronFortificationEffect:
+
+    @property
+    def name(self):
+        return 'maternal_iron_fortification_effect'
+
+    def setup(self, builder: 'Builder'):
+        self.treatment_effects = self.load_treatment_effects(builder)
+
+        builder.value.register_value_modifier(f'low_birth_weight_and_short_gestation.exposure',
```

Comment:
Trace through and make sure this is the right place to modify.  Use examples from BEP.

Reply:
Maternal bmi and ?